include/libswiftnav/nav_msg.h

```diff
@@ -16,11 +16,12 @@
 #include "common.h"
 #include "ephemeris.h"
 
-#define NAV_MSG_SUBFRAME_BITS_LEN 12 /* Buffer 384 nav bits. */
+#define NAV_MSG_SUBFRAME_BITS_LEN 14 /* Buffer 448 nav bits. */
```

Comment:
Feel free to increase further if you think it would be beneficial. Obviously we should fix the priorities as well.